composer.json

```diff
@@ -1,12 +1,13 @@
 {
-	"name": "webspell/webspell",
-	"description": "webSPELL CMS",
-	"keywords": ["cms", "gaming"],
-	"license": "GPL-3.0",
-	"type": "project",
-	"require": {
-		"squizlabs/php_codesniffer": "2.1.0",
-		"codeclimate/php-test-reporter": "dev-master",
-		"phpunit/phpunit": "4.4.0"
-	}
+    "name": "webspell/webspell",
+    "description": "webSPELL CMS",
+    "keywords": ["cms", "gaming"],
+    "license": "GPL-3.0",
+    "type": "project",
+    "require": {
+        "squizlabs/php_codesniffer": "2.1.0",
+        "codeclimate/php-test-reporter": "dev-master",
+        "phpunit/phpunit": "4.4.0",
+        "phpmailer/phpmailer": "5.2.*"
```

Comment:
This should be the only `require` element, all codesniffer, test-reporter and phpunit are [`require-dev`](https://getcomposer.org/doc/04-schema.md#require-dev).